src/cljs/main/broadfcui/components/tab_bar.cljs

```diff
@@ -11,7 +11,7 @@
    (fn [{:keys [props state]}]
      (let [{:keys [data-test-id first? active? label href on-refresh]} props]
        [:a {:data-test-id data-test-id
-            :style {:flex "0 0 auto" :padding "1em 2em"
+            :style {:flex "0 0 auto" :padding (if (= label "Notebooks") ".70em 2em" "1em 2em")
```

Comment:
Can you add a comment that this is for the beta label? (I don't know what conventions are around comments for you all, but I could see this potentially being confusing down the line.)